README.md

```diff
@@ -1 +1,142 @@
-# ToDoList2
\ No newline at end of file
+<a name="readme-top"></a>
+
+all the comments and instructions!
+
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [OPTIONAL] I noticed that this image link is broken, this is a placeholder logo added by Microverse, I'll suggest you remove this as it's not needed in your readme and the broken logo makes your readme look a bit unprofessional. :smile: :airplane: